### PR TITLE
[RW-6125][risk=low]: Remove egress alert; add more README

### PR DESCRIPTION
**Pull request by yonghaoy**
This PR does 2 things:

1. Remove egress alert because it should be done in terraform-terra repo
2. Add instruction on where to pull monitoring modules.

This should only affect test&sandbox&local env

### CODE REVIEW

**Review by calbach — COMMENTED** [review]
Wait - why would we move this to the terraform-terra repo? I'm confused. You want to move sumologic configuration there as well?

**Comment by yonghaoy** [comment]
> Wait - why would we move this to the terraform-terra repo? I'm confused. You want to move sumologic configuration there as well?

They are already there:
https://github.com/broadinstitute/terraform-terra/blob/master/profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/vpc-flow-egress-alert.tf

We think this is better to have the whole process(pubsub, IAM, logsink, gcp source, sumologic setup) as part of service perimeter creation.

File: README.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ We use Terraform to manage metrics descriptor, logs-based Metrics, alert policie
 To add/edit a new one: you can either add/edit on json files directly or manually setup in GCP console to get json file.
 One recommended way can be:
 1. Manually create/modify metrics/alerts/dahsboard in test env. 
-2. Get the updated json file via google API or gcloud console.
+2. Get the updated json file via google API or gcloud console. Api explorer:
+    * [Lists metric descriptors](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors/list?apix_params=%7B%22name%22%3A%22projects%2Fall-of-us-rw-prod%22%2C%22filter%22%3A%22metric.type%20%3D%20starts_with(%5C%22logging.googleapis.com%2Fuser%5C%22)%22%7D)
```
Comment (calbach) [inline README.md:53]: nit: A gcloud command would be better IMO (list, then describe), unless no such command exists

(e.g. `gcloud --format json ...`)

**Review by calbach — APPROVED** [review]
Hm.. not sure about this. From a setup automation perspective, it makes sense. However, from an application/ownership perspective - the alerts themselves are pretty tightly tied to the AoU applications, and we want to be able to control the thresholds ourselves. But I guess it's a bit on the edge, since the Sumologic instance belongs to the Broad anyways.

I suppose I'm fine with this if it's the path you've already started down. Also definitely a bonus that we don't need to mess with Sumologic keys to apply Terraform changes after this.

Is there a third item missing from the list? `This PR does three things:`

**Comment by yonghaoy** [comment]
> Hm.. not sure about this. From a setup automation perspective, it makes sense. However, from an application/ownership perspective - the alerts themselves are pretty tightly tied to the AoU applications, and we want to be able to control the thresholds ourselves. But I guess it's a bit on the edge, since the Sumologic instance belongs to the Broad anyways.
> 
> I suppose I'm fine with this if it's the path you've already started down. Also definitely a bonus that we don't need to mess with Sumologic keys to apply Terraform changes after this.
> 
> Is there a third item missing from the list? `This PR does three things:`

Or we can understand that sumologic is: Setting up sumologic egress alert per service premeter. 
I treat it is the same as the exsiting ingress-bridges.tf, log-sink.tf. 
(It setsup something for a Rawls Service perimeters, but only enabled for AoU). Moving forward, the sumo forlder, threshold all configuratble. 

Sorry, now PR only does two things.

**Comment by calbach** [comment]
I understand, though right now the alert structure and the query are pretty AoU-specific, and would need to be generalized to support a second consumer, e.g. https://github.com/broadinstitute/terraform-terra/blob/master/profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/query.txt#L39 , https://github.com/broadinstitute/terraform-terra/blob/master/profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/query.txt#L77-L89

File: README.md (hunk 1 above)
Comment (yonghaoy) [reply README.md:53]: Found corresponding `gcloud` command for alert, logging, and dashboard, couldn't find metrics descriptor. 
Not even in alpha: https://cloud.google.com/sdk/gcloud/reference/alpha/monitoring

Comment (calbach) [reply README.md:53]: ack, thanks for checking

**Review by calbach — APPROVED** [review]

**Pull request merged**